### PR TITLE
re-introduce log message for link creation

### PR DESCRIPTION
**Pull request by steiler**

### CODE REVIEW

**Comment by steiler** [comment]
@hellt this is the log message pr

File: links/link_veth.go [hunk 1]
```diff
@@ -6,6 +6,7 @@ import (
 	"sync"
 
 	"github.com/containernetworking/plugins/pkg/ns"
+	"github.com/sirupsen/logrus"
```
Comment (hellt) [inline links/link_veth.go]:
```suggestion
	log "github.com/sirupsen/logrus"
```

File: links/link_macvlan.go [hunk 2]
```diff
@@ -138,6 +139,8 @@ func (l *LinkMacVlan) GetParentInterfaceMtu() (int, error) {
 }
 
 func (l *LinkMacVlan) Deploy(ctx context.Context) error {
+	log.Infof("Creating virtual wire (MacVlan): %s <--> %s", l.HostEndpoint.String(), l.NodeEndpoint.String())
```
Comment (hellt) [inline links/link_macvlan.go]:
```suggestion
	log.Infof("Creating MACVLAN link: %s <--> %s", l.HostEndpoint.String(), l.NodeEndpoint.String())
```

File: links/link_veth.go [hunk 3]
```diff
@@ -106,6 +107,8 @@ func (l *LinkVEth) Verify() {
 }
 
 func (l *LinkVEth) Deploy(ctx context.Context) error {
+	logrus.Infof("Creating virtual wire (veth): %s <--> %s", l.GetEndpoints()[0].String(), l.GetEndpoints()[1].String())
```
Comment (hellt) [inline links/link_veth.go]:
```suggestion
	log.Infof("Creating link: %s <--> %s", l.GetEndpoints()[0].String(), l.GetEndpoints()[1].String())
```

**Comment by steiler** [comment]
@hellt this can also be merged now, correct?

**Comment by hellt** [comment]
@steiler yes

**Pull request merged**